apps/src/codebridge/Workspace/Workspace.tsx

```diff
@@ -69,7 +89,16 @@ const Workspace: React.FunctionComponent<WorkspaceProps> = ({
         {viewingOldVersion && (
           <Alert
             className={moduleStyles.previousVersionBanner}
-            text={codebridgeI18n.viewingOldVersion()}
+            text={
```

Comment:
Nit: this is getting a bit busy here so could assign `text` to a variable like `versionBannerText` defined earlier in component.

```
const versionBannerText = versionDate ? (
  <>
    You're viewing a previous version of this project from{' '}
    <strong>{versionDate}</strong>.
  </>
) : (
  "You're viewing the initial version of this project."
);
```